daemon/archive.go

```diff
@@ -155,9 +156,15 @@ func (container *Container) ArchivePath(path string) (content io.ReadCloser, sta
 	}
 
 	// Consider the given path as an absolute path in the container.
-	absPath := path
-	if !filepath.IsAbs(absPath) {
-		absPath = archive.PreserveTrailingDotOrSeparator(filepath.Join(string(os.PathSeparator), path), path)
+	absPath := srcpath
+
+	// Cross-platform note. We CANNOT use filepath here as on Windows that
+	// would check for drive-letters (eg c:\windows) to indicate absoluteness.
+	// However, the path here on Windows may legitimately be \windows which IS
+	// absolute. The answer is to ToSlash and use path.IsAbs, not filepath.IsAbs.
+	// This is OK on Linux platforms, as ToSlash will be a no-op.
+	if !path.IsAbs(filepath.ToSlash(absPath)) {
```

Comment:
An absolute path on Windows includes a drive letter. However, because of us using Linux path semantics, an absolute path in docker cp of /windows (\windows) must also be treated as absolute. Shorter answer, no :)